### PR TITLE
Small Renovation 

### PR DESCRIPTION
**Pull request by kerrtravers**
## New Features
- Adds transparent Favicon
- Updates wording slightly 

## Reasons for this update
- Looks a lot better for anyone using dark browser themes 
- Easier to understand for younger users

### CODE REVIEW

File: global-stats.html [hunk 1]
```diff
@@ -274,7 +274,7 @@ <h3 class="noBottomMargin">Most Followed Scratchers</h3>
     </div>
     <br><br><br>
     <div class="text-block statstable" id="mostPopularProjects">
-      <h3 class="noBottomMargin">Most Popular Projects <small>(source: Explore)</small></h3>
+      <h3 class="noBottomMargin">Most Popular Projects <small>(From Explore)</small></h3>
```
Comment (WorldLanguages) [inline global-stats.html:277]: Nice.

**Comment by WorldLanguages** [comment]
Thanks!

**Comment by kerrtravers** [comment]
No Problem 💻

**Comment by Sheshank-s** [comment]
favicon isn't transparent ill try to fix.

**Comment by Sheshank-s** [comment]
@WorldLanguages Idk how to fix xD

**Comment by kerrtravers** [comment]
@GreenBayRules Really? It looks fine to me...
capture

Btw It's a transparent background, Not a transparent logo. It just removes the white bit from the favicon.

**Comment by Sheshank-s** [comment]
hmmm ill send you screenshot when I get home @VutonDesign But not working for me.

**Comment by kerrtravers** [comment]
Okay, Try pressing Crtl + f5 to clear you cache?

**Comment by Sheshank-s** [comment]
image

**Comment by kerrtravers** [comment]
Did you clear your browser history and cache?
Looks like this to me:
capture

**Comment by Sheshank-s** [comment]
yep cleared the cache.

**Comment by WorldLanguages** [comment]
Go here http://scratchstats.cf/favicon.ico press ctrl+f5 @GreenBayRules

**Comment by kerrtravers** [comment]
^ @GreenBayRules That's what works for me.

**Pull request merged**